cmd/zoekt-sourcegraph-indexserver/merge.go

```diff
@@ -0,0 +1,237 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/zoekt"
+)
+
+// doMerge drives the merge process, I imagine this function goes away once we
+// automate merging.
+func doMerge(params string) error {
+	dir, maxSize, targetSize, days, simulate, err := parseParams(params)
+	if err != nil {
+		return err
+	}
+
+	if simulate {
+		debug.Println("simulating")
+	}
+
+	shards := loadShards(dir)
+	if len(shards) == 0 {
+		return fmt.Errorf("no shards found")
+	}
+	debug.Printf("found %d shards\n", len(shards))
+
+	backupDir := filepath.Join(dir, ".scratch/bak")
```

Comment:
I removed the backup and simply delete the shards. Since we don't merge recent or very important shards we can simply reindex them in case something goes wrong. WDYT?